src/drive/styles/layout.styl

```diff
@@ -2,6 +2,8 @@
 
 .fil-wrapper
     @extend    $app-2panes-sticky
+    padding-left env(safe-area-inset-left)
 
 .fil-sidebar
     justify-content  space-between
+    padding-bottom env(safe-area-inset-bottom)
```

Comment:
@GoOz I think it would be good to add these changes to cozy-ui, but I'm not familiar with the different layouts these. [More infos here](https://webkit.org/blog/7929/designing-websites-for-iphone-x/).

Reply:
`safe-area-inset-bottom` is only provided on the iphone x, on anything else it just produces an invalid rule that is ignored :)

Reply:
Yeah I just saw your link and it took me time to read it. Sorry, we're good then. ;)